pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerCloseTest.java

```diff
@@ -88,14 +88,18 @@ private ServiceConfiguration brokerConfig() {
         config.setLoadManagerClassName(ExtensibleLoadManagerImpl.class.getName());
         config.setLoadBalancerDebugModeEnabled(true);
         config.setBrokerShutdownTimeoutMs(100);
+
+        // Reduce these timeout configs to avoid failed tests being blocked too long
+        config.setMetadataStoreOperationTimeoutSeconds(5);
+        config.setNamespaceBundleUnloadingTimeoutMs(5000);
         return config;
     }
 
 
-    @Test
+    @Test(invocationCount = 10)
```

Comment:
We should consider adding a new unit test group solely for all load manager related tests. That could be handled separately. I'm planning to add a new group for dispatcher tests since the main Key_Shared test takes almost 10 minutes even on Mac M3Max locally. The test time doubled since now we test also the "classic" dispatcher for Key_Shared that was copied and ported from branch-3.3 in #23424

Reply:
Agreed. The test groups need to be re-assigned.